src/natcap/invest/sdr/sdr.py

```diff
@@ -147,6 +147,169 @@
+                },
+                "ic.tif": {
+                    "about": "index of connectivity (Eq. (70))"
+                },
+                "ls.tif": {
+                    "about": "LS factor for USLE (Eq. (69))"
+                },
+                "pit_filled_dem.tif": {
+                    "about": "DEM after any pits are filled"
+                },
+                "s_accumulation.tif": {
+                    "about": "flow accumulation weighted by the thresholded slope. Used in calculating s_bar."
+                },
+                "s_bar.tif": {
+                    "about": "mean thresholded slope gradient of the upslope contributing area (in eq. (73))"
+                },
+                "s_inverse.tif": {
+                    "about": "inverse of the thresholded slope (in eq. (74))"
+                },
+                "sdr_factor.tif": {
+                    "about": "sediment delivery ratio (Eq. (75))"
+                },
+                "slope.tif": {
+                    "about": "slope in radians, calculated from the pit-filled DEM"
+                },
+                "slope_threshold.tif": {
+                    "about": "slope, thresholded to be no less than 0.005 and no greater than 1 (eq. (71))",
+                    "bands": {1: {
+                        "type": "number",
+                        "units": u.radian
+                    }}
+                },
+                "w_threshold.tif": {
+                    "about": "cover-management factor thresholded to be no less than 0.001 (eq. (72))"
+                },
+                "w_accumulation.tif": {
+                    "about": "flow accumulation weighted by the thresholded cover-management factor. Used in calculating w_bar."
+                },
+                "w_bar.tif": {
+                    "about": "mean thresholded cover-management factor for upslope contributing area (in eq. (73))"
+                },
+                "w.tif": {
+                    "about": "cover-management factor derived by mapping usle_c from the biophysical table to the LULC raster"
+                },
+                "what_drains_to_stream.tif": {
+                    "about": "Map of which pixels drain to a stream. A value of 1 means that at least some of the runoff from that pixel drains to a stream in stream.tif. A value of 0 means that it does not drain at all to any stream in stream.tif."
+                },
+                "weighted_avg_aspect.tif": {
+                    "about": "average aspect weighted by flow direction(in eq. (69))"
+                },
+                "ws_inverse.tif": {
+                    "about": "Inverse of the thresholded cover-management factor times the thresholded slope (in eq. (74))"
+                },
+                "churn_dir_not_for_humans": spec_utils.TASKGRAPH_DIR
```

Comment:
I think SDR is somewhat unique in that this taskgraph directory also contains some preprocessing files.  Should those also be included here?